packages/cf-component-heading/src/HeadingCaptionTheme.js

```diff
@@ -0,0 +1,7 @@
+export default () => ({
+  color: '#434148',
```

Comment:
We need to move all these variables into a const file early before we start putting hex codes everywhere.  @tajo does this exist yet?